src/main/java/com/example/simplefileshare/simplefileshare/services/utils/MultipartFileToJavaFileUtil.java

```
@@ -0,0 +1,22 @@
+package com.example.simplefileshare.simplefileshare.services.utils;
+import org.springframework.stereotype.Service;
+import org.springframework.web.multipart.MultipartFile;
+import java.io.FileNotFoundException;
+import java.io.File;
+
+@Service
+public class MultipartFileToJavaFileUtil {
+    
+    public File convert(MultipartFile multipartFile) throws FileNotFoundException {
+        try{
+            File file = new File(multipartFile.getName());
+            multipartFile.transferTo(file);
+            System.out.println("File Created Successfully");
+            return file;
+        }
+        catch(Exception e){
```

Can we throw the error exactly as it is and reserve FileNotFoundException to only those cases where file is not found in the request?

---

okay will add it.